src/SourceBuild/content/repo-projects/runtime.proj

```diff
@@ -29,6 +29,9 @@
 
     <BuildArgs Condition="'$(UseSystemLibs)' != ''">$(BuildArgs) /p:UseSystemLibs=$(UseSystemLibs)</BuildArgs>
 
+    <!-- When we're actually doing signing and the ESRP tool is available, forward down the path to the repo builds. -->
+    <BuildArgs Condition="'$(Sign)' == 'true' and '$(ForceDryRunSigning)' == '' and '$(DotNetEsrpToolPath)' != ''">$(BuildArgs) /p:DotNetEsrpToolPath=$(DotNetEsrpToolPath)</BuildArgs>
```

Comment:
Shouldn't this be `'$(ForceDryRunSigning)' != 'true'` so that this also kicks in when YML or someone explicitly passes /p:ForceDryRunSigning=false in?

Reply:
```suggestion
    <BuildArgs Condition="'$(Sign)' == 'true' and '$(ForceDryRunSigning)' != 'true' and '$(DotNetEsrpToolPath)' != ''">$(BuildArgs) /p:DotNetEsrpToolPath=$(DotNetEsrpToolPath)</BuildArgs>
```